profiler_measurements/highDroppingRateScenario.h

```diff
@@ -0,0 +1,71 @@
+#ifndef PIPERT_HIGHDROPPING_SCENARIO_H
+#define PIPERT_HIGHDROPPING_SCENARIO_H
+#include <fcntl.h>
+#include <termio.h>
+#include <unistd.h>
+
+#include <algorithm>
```

Comment:
Chekc which headers are really needed as giving more then enough will slow down the compilation.